src/org/usfirst/frc/team467/robot/Grabber.java

```diff
@@ -63,12 +70,24 @@ public void periodic() {
 
 		double speed = 0.0;
 		switch (state) {
+		
+		case START_GRAB:
+			if(hasCube()) {
+				state = GrabberState.NEUTRAL;
+			}
+			else {
+				speed = RobotMap.MAX_GRAB_SPEED;
+			}
+			break;
 
 		case GRAB:
-			if(hasCube() || count > grabCount) {
+			if(hasCube()) { 
 				state = GrabberState.NEUTRAL;
+				//count = 0;
 			} else {
 				speed = RobotMap.MAX_GRAB_SPEED;
+				drive.moveLinearFeet(2.08);     // tester
```

Comment:
This can't be right. !! You shouldn't be driving from this class. What if you wanted to grab and drive a different distance?